src/components/action-popover/action-popover.style.ts

```diff
@@ -105,9 +105,17 @@ const StyledButtonIcon = styled.div`
   }
 `;
 
-const MenuItemIcon = styled(Icon)`
-  padding: var(--spacing100);
+const MenuItemIcon = styled(Icon)<StyledMenuItemProps>`
```

Comment:
I'm not sure I like the use of `StyledMenuItemProps` here, as the props aren't the same - `StyledMenuItemProps` also has `isDisabled` which has nothing to do with `MenuItemIcon`.

Better imo would be to type it as either `{ horizontalAlignment: "left" | "right" }`, or perhaps `Pick<StyledMenuItemProps, "horizontalAlignment">` if we're sure that the two styled components should always have the same type for that prop (which they probably should). I'm also happy with making an alias (`StyledMenuItemIconProps` or similar) for either of these options (not too bothered either way) - but I don't think it's right to use the whole of `StyledMenuItemProps`.